From fa4bea7df2e8d12f885fcc2ee14907d8f48cd603 Mon Sep 17 00:00:00 2001
From: Miao Yu <r65093@freescale.com>
Date: Tue, 3 Nov 2009 13:03:24 +0800
Subject: [PATCH 125/505] ENGR00117975 sahara2: Fix compile error when enable
 armv7a build flag

If enable armv7-a when using code sourcery gcc 4.3.3 toolchain,
imx-lib will encounter build error when compiling sahara2.

Signed-off-by: Miao Yu <r65093@freescale.com>
---
 test/mxc_sahara_test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_sahara_test/Makefile b/test/mxc_sahara_test/Makefile
index 5a92736..5777a8c 100644
--- a/test/mxc_sahara_test/Makefile
+++ b/test/mxc_sahara_test/Makefile
@@ -12,7 +12,7 @@ TARGET_ARCH=-DCONFIG_ARCH_MX51
 endif
 
 # Add compilation checks
-CFLAGS += -g -Wall -W -Wmissing-prototypes -Wstrict-prototypes -Wdeclaration-after-statement
+CFLAGS += -g -Wall -W -Wmissing-prototypes -Wstrict-prototypes -Wdeclaration-after-statement -fPIC
 
 EXAMPLE_OBJECTS:= apitest.o run_tests.o sym.o hash.o gen_encrypt.o \
                   callback.o auth_decrypt.o rand.o hmac1.o hmac2.o cap.o \
-- 
1.8.0

